docs/tangle.md

```diff
@@ -0,0 +1,276 @@
+        <td>Sequence number</td>
+        <td>uint64</td>
+        <td>The always increasing number of issued messages of the issuing node.</td>
+    </tr>
+    <tr>
+        <td>Payload length</td>
+        <td>uint32</td>
+        <td>The length of the Payload. Since its type may be unknown to the node it must be declared in advance. 0 length means no payload will be attached.</td>
+    </tr>
+    <tr>
+        <td colspan="1">
+            Payload
+        </td>
+        <td colspan="2">
+            <details open="true">
+                <summary>Generic Payload</summary>
+                <blockquote>
+                An outline of a general payload
+                </blockquote>
+                <table>
+                    <tr>
+                        <th>Name</th>
+                        <th>Type</th>
+                        <th>Description</th>
+                    </tr>
+                    <tr>
+                        <td>Payload Type</td>
+                        <td>uint32</td>
+                        <td>
+                            The type of the payload. It will instruct the node how to parse the fields that follow. Types in the range of 0-127 are "core types" that all nodes are expected to know.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Data Fields</td>
+                        <td>ANY</td>
+                        <td>A sequence of fields, where the structure depends on <code>payload type</code>.</td>
+                    </tr>
+                </table>
+            </details>
+            </td>
+    </tr>
+    <tr>
+        <td>Nonce</td>
+        <td>uint64</td>
+        <td>The nonce which lets this message fulfill the Proof-of-Work requirement.</td>
+    </tr>
+    <tr>
+        <td>Signature (Ed25519)</td>
+        <td>ByteArray[64]</td>
+        <td>Signature of the issuing node's private key signing the entire message bytes.</td>
+    </tr>
+</table>
+
+
+### Syntactical Validation
+1. The message length must not exceed `MAX_MESSAGE_SIZE` bytes.
+2. When we are done parsing the message there shouldn't be any trailing bytes left that were not parsed.
+4. At least 1 and maximum 8 distinct parents are given, ordered ASC and at least `MIN_STRONG_PARENTS` are strong parent. 
+
+### Semantic Validation / Eligibility
+Only eligible messages are processed by a node, added to its local Tangle and thus can become available as tips for tip selection. Eligibility of a message **does not** express any opinion about the attachment location of a message. It solely evaluates a message according to its timestamp.
+
+A message is an eligible message if:
+- If the Message PoW Hash will contain at least the number of leading 0 the node defines as required.
+-  Verify signature to be valid from issuing node.
+- It is solid
+- It has a good timestamp (TODO: link timestamp spec)
+- It passes parents age checks (TODO: link timestamp spec)
+- Its parents are eligible
+
```

Comment:
By only reading this part of the spec, it is not clear if the payload can affect the eligibility for TSA. It sounds like 
good timestamp => available for TSA
Maybe we can add a comment about this?

Reply:
Also, it's not super clear how the eligibility for TSA and the weak/strong parents are related (but maybe this is not the right spec to talk about this? I'm not sure...). For instance, eligible for TSA means eligible to become a strong parent, right? 
I guess what is missing actually is the distinction between not being eligible because of the timestamp of the message and not being eligible because of the payload